config/webpack-config.js

```diff
@@ -27,13 +27,11 @@ module.exports = {
   plugins: [
     new webpack.optimize.CommonsChunkPlugin( COMMON_BUNDLE_NAME ),
     new webpack.optimize.CommonsChunkPlugin( COMMON_BUNDLE_NAME,
-                                             [ COMMON_BUNDLE_NAME ] ),
+                                             'common.[chunkhash].js' ),
```

Comment:
So how's this working now? It generates `common.js` and `common.*.js` that are duplicates of each other, is that right? Can we just spit out the hashed version?

One consequence of this I'm seeing is `/static_built/` starts to build up a number of hashed common.js files. We may want to include a step in backend.sh or frontend.sh to flush out that directory, if that doesn't currently happen.